pkg/ride/functions_bigint.go

```diff
@@ -104,16 +104,16 @@ func powBigInt(_ Environment, args ...rideType) (rideType, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "powBigInt")
 	}
-	b := big.NewInt(0).Set(base.v)
-	e := big.NewInt(0).Set(exponent.v)
+	b := big.NewInt(0).Set(base.V)
```

Comment:
Why `V` is public in private structure? Looks like somebody forgot to change it back from public to private.